docs/source/usage.rst

```diff
@@ -23,3 +23,29 @@ The evaluation of submissions are conducted on a Docker image that is built whil
 If you are very conservative about populating your default environment with random Python packages, create a virtual environment for installing some *new* packages either using ``virtualenv`` or ``conda-env``.
 
 Install the requirements specified in `requirements.txt <../../../requirements.txt>`_. Don't forget to activate your environment if you have one.
+
+Phase 2 : Run ``autojudge``
+---------------------------
+
+Activate your environment. 
+
+Create and apply database migrations in Django with the following commands:
+
+.. code:: bash
+
+    python manage.py makemigrations
+    python manage.py migrate
+
+To run the judge locally:
+
+.. code:: bash
+
+    python manage.py runserver
```

Comment:
Optional IP is missing.

Reply:
@prateekkumarweb is going to elucidate this part in detail for deployment / local testing.